commands/members/add.js

```diff
@@ -5,16 +5,25 @@ let co     = require('co');
 let extend = require('util')._extend;
 
 function* run (context, heroku) {
+  const FREE_TEAM_LIMIT = 5;
   let org   = context.org;
   let email = context.args.email;
   let role  = context.flags.role;
+  let members = yield heroku.get(`/organizations/${org}/members`);
+  let features = yield heroku.get('/account/features');
+  let orgCreationFeature = features.find(function(feature) {
```

Comment:
Oh, right!. It doesn't look like you can with `_.includes`, so maybe what you've got works better :P